spec/struct.dd

```diff
@@ -1373,6 +1373,237 @@ $(H3 $(LNAME2 implicit-copy-constructors, Implicit Copy Constructors))
 
     $(P If the generated copy constructor fails to type check, it will receive the `@disable` attribute.)
 
+$(H2 $(LEGACY_LNAME2 StructMoveConstructor, struct-move-constructor, Struct Move Constructors))
+
+    $(P Move constructors are very much like copy constructors. The difference is that copy
+    constructors make a copy of the original, while move constructors move the contents of
+    the original, and the lifetime of the original ends.)
+
+    $(NOTE Do not use postblits in the same struct with move constructors.)
+
+    $(COMMENT $(P A `struct` that defines a move constructor
+    is not $(RELATIVE_LINK2 POD, POD).))
```

Comment:
Just to check, was this intended to appear in the output? (The COMMENT macro is ignored).

Reply:
COMMENT macros are ignored on purpose.